kubernetes/charts/weblogic-domain/templates/_domain-custom-resource.tpl

```diff
@@ -27,6 +27,9 @@ spec:
   image: "{{ .weblogicImage }}"
   # imagePullPolicy defaults to "Always" if image version is :latest
   imagePullPolicy: "IfNotPresent"
+  # Identify which Secret containst the credentials to access the docker repository
```

Comment:
typo - containst

Might not be important to fix since we're planning on removing the domain helm chart.